packages/host/app/helpers/array-helpers.ts

```diff
@@ -0,0 +1,5 @@
+export function reverse<T>(arr: T[]): T[] {
+  //Avoids error when reverse directly on tracked array
```

Comment:
Can you add a note about which error specifically?

Reply:
Can this be a getter instead?

Reply:
Updated.